app-server/nginx+RR/Dockerfile

```diff
@@ -1,11 +1,10 @@
-FROM --platform=${TARGETPLATFORM:-linux/amd64} ghcr.io/roadrunner-server/roadrunner:latest as roadrunner
-FROM --platform=${TARGETPLATFORM:-linux/amd64} php:8.1-alpine
+FROM --platform=${TARGETPLATFORM:-linux/amd64} ghcr.io/roadrunner-server/roadrunner:2024 as roadrunner
+FROM --platform=${TARGETPLATFORM:-linux/amd64} php:8.3-alpine
```

Comment:
I think we can safely use `latest` here. Just don't want to change the version every year 😄

Reply:
`latest` itself is not a bad practice, it's just a tag. Bad practice is to use `latest` in production, and that's it.
Let's keep it clear, I do want to update docs, and I'm doing that regularly, the problem is, that that's not possible to maintain a version in one place, like some constant. Thus, please, use `latest` tag.

Reply:
Done.